it/docker-compose.yml

```diff
@@ -7,6 +7,8 @@ services:
       - "9000:9000"
     environment:
       ES_JAVA_OPTS: "-Xms750m -Xmx750m"
+    security_opt:
+      - seccomp:unconfined
```

Comment:
Required because of https://github.com/SonarSource/docker-sonarqube/issues/493

Reply:
When it will be released in docker hub?